docker/web/production/docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+service cron start
+./web orm:generate-proxies
+memcached -m 256 -p 11211 -U 11211 -u www-data -d
```

Comment:
What happens if memcached crashes? The container won't restart because memcached is not running in the foreground.

I think we should either do it the docker way and run everything in separate containers OR just go ahead and run everything in a single container and run something like supervisor in it as well to make sure everything keeps running.

Reply:
I agree. I will move memcached to a seperate container